src/utils/Cargo.toml

```diff
@@ -13,6 +13,7 @@ thiserror = "1.0.32"
 versionize = "0.1.10"
 versionize_derive = "0.1.5"
 vmm-sys-util = "0.11.0"
+vm-memory-upstream = { package = "vm-memory",  version = "0.10.0", features = ["backend-mmap", "backend-bitmap"] }
```

Comment:
Mh, since there is no "vm_memory" crate in the workspace anymore, do we still need to rename the dependency?

Reply:
Done. Renamed the upstream create. Please re-review.